bundles/org.openhab.io.homekit/src/main/java/org/openhab/io/homekit/internal/HomekitCommandExtension.java

```diff
@@ -127,8 +116,7 @@ public void unsetHomekit(Homekit homekit) {
 
     private void clearHomekitPairings(Console console) {
         try {
-            new HomekitAuthInfoImpl(storageService, null).clear();
-            homekit.refreshAuthInfo();
+            homekit.clearHomekitPairings();
             console.println("Cleared HomeKit pairings");
         } catch (Exception e) {
```

Comment:
You should be able to remove this try/catch now

Reply:
right. removed.